cg/apps/demultiplex/sample_sheet/models.py

```diff
@@ -66,6 +66,15 @@ class FlowCellSampleBCLConvert(FlowCellSample):
 class SampleSheet(BaseModel):
     samples: List[FlowCellSample]
 
+    def get_non_pooled_lanes_and_samples(self) -> Iterable[Tuple[int, str]]:
+        """Return tuples of non pooled lane and sample ids."""
+        non_pooled_lane_sample_id_pairs: List[Tuple[int, str]] = []
+        non_pooled_samples: List[FlowCellSample] = self.get_non_pooled_samples()
+        for sample in non_pooled_samples:
+            sample_id: str = sample.sample_id.split("_")[0]
```

Comment:
is this not the same as the "get_valid_sample_ids" function that was implemented in a recent pr? Maybe we can re-use that here.

Reply:
It is also in here: https://github.com/Clinical-Genomics/cg/pull/2491/files

Reply:
Yes, that code has not been merged yet. Can update it once this is merged 👍